pnpm-lock.yaml

```diff
@@ -2146,15 +2146,15 @@ packages:
       set-cookie-parser: 2.5.1
     dev: true
 
-  /@mswjs/interceptors/0.17.4:
-    resolution: {integrity: sha512-8oKWrOQ1P0Wj0kf3ak8WETuymknw2Tl2s1op8OzZctpNF3zSJSdEbcQs0pm347mwsM+95V6POBMv3W4812pEeg==}
+  /@mswjs/interceptors/0.17.5:
```

Comment:
@HonkingGoose Do you happen to know why this is getting updated if I added it to the `ignoreDeps` in the renovate config file?

Reply:
> @HonkingGoose Do you happen to know why this is getting updated if I added it to the ignoreDeps in the renovate config file?

@Belco90 I guess that Renovate lets the _package manager_ deal with lockfile maintenance. So Renovate itself doesn't "touch" the lockfile.

The docs for `ignoreDeps` don't mention lockfile-maintenance. So let's see what the Renovate devs say here:

- https://github.com/renovatebot/renovate/discussions/17982

Reply:
Thank you!

Reply:
All good now using pnpm overrides 😎